### PR TITLE
[CAP 35] Add happy-path integrationt test for ClawbackClaimableBalance

### PR DESCRIPTION
**Pull request by 2opremio**
Part of #3348

### CODE REVIEW

File: services/horizon/internal/integration/protocol16_test.go [hunk 1]
```diff
@@ -154,3 +153,101 @@ func TestHappyClawback(t *testing.T) {
...
+	itest := integration.NewTest(t, protocol16Config)
+	master := itest.Master()
+
+	// Give the master account the revocable flag (needed to set the clawback flag)
+	setRevocableFlag := txnbuild.SetOptions{
+		SetFlags: []txnbuild.AccountFlag{
+			txnbuild.AuthRevocable,
+		},
+	}
+	_, err := itest.SubmitOperations(itest.MasterAccount(), master, &setRevocableFlag)
```
Comment (bartekn) [inline services/horizon/internal/integration/protocol16_test.go:168]: Nit
```suggestion
	itest.MustSubmitOperations(itest.MasterAccount(), master, &setRevocableFlag)
```

File: services/horizon/internal/integration/protocol16_test.go [hunk 2]
```diff
@@ -154,3 +153,101 @@ func TestHappyClawback(t *testing.T) {
...
+			txnbuild.AuthClawbackEnabled,
+		},
+	}
+	_, err = itest.SubmitOperations(itest.MasterAccount(), master, &setClawBackFlag)
+	tt.NoError(err)
+
+	// Make sure the clawback flag was set
+	accountDetails, err := itest.Client().AccountDetail(horizonclient.AccountRequest{
+		AccountID: master.Address(),
+	})
```
Comment (bartekn) [inline services/horizon/internal/integration/protocol16_test.go:183]:
```suggestion
	accountDetails, err := itest.Client().AccountDetail(horizonclient.AccountRequest{
		AccountID: master.Address(),
	})
	tt.NoError(err)
```

File: services/horizon/internal/integration/protocol16_test.go [hunk 3]
```diff
@@ -154,3 +153,101 @@ func TestHappyClawback(t *testing.T) {
...
+			txnbuild.AuthRevocable,
+		},
+	}
+	_, err := itest.SubmitOperations(itest.MasterAccount(), master, &setRevocableFlag)
+	tt.NoError(err)
+
+	// Give the master account the clawback flag
+	setClawBackFlag := txnbuild.SetOptions{
+		SetFlags: []txnbuild.AccountFlag{
+			txnbuild.AuthClawbackEnabled,
```
Comment (bartekn) [inline services/horizon/internal/integration/protocol16_test.go:174]: I think the flag is `true` by default so maybe it's not necessary.

**Review by bartekn — APPROVED** [review]

File: services/horizon/internal/integration/protocol16_test.go (hunk 3 above)
Comment (2opremio) [reply services/horizon/internal/integration/protocol16_test.go:174]: It is necessary (I had to add it in the clawback test)

**Pull request merged**